core/math/aabb.h

```diff
@@ -41,6 +41,11 @@
 class Variant;
 
 struct [[nodiscard]] AABB {
+	static const AABB ZERO;
+	static const AABB ONE;
+	static const AABB INF;
+	static const AABB NaN;
```

Comment:
Note: In order to make these constants `constexpr`, they had to first be declared in the structs as `const`. Attempting to define them as `constexpr` at this stage raises an incomplete-type error